src/test/lib/TestConnection.h

```diff
@@ -304,6 +304,8 @@ class TestConnection
 
     QUIC_STATUS SetResumptionTicket(const QUIC_BUFFER* ResumptionTicket) const;
 
+    QUIC_STATUS SetQtipPreferences(const uint8_t OnOrOff) const;
```

Comment:
nit: `const` on a by-value parameter in a prototype should be removed (it is totally ok to have it in the definition but not the declaration).

Reply:
ACK, fix next iteration.